### PR TITLE
bootstrap tooltips for location tags

### LINKED ISSUE #9732: [Discussion] Add Bootstrap Tooltips to Location Tags
The [description](https://github.com/publiclab/plots2/issues/9257) for the location tag tooltips task says to add the tooltips to a page like https://publiclab.org/wiki/puerto-rico but I can't find any location tags on that page that need tooltips. The location tags I did find already have tooltips 

Screenshot 2021-06-03 at 12 17 05

Should it be in the location tags in the profile page https://publiclab.org/profile/mathildaudufo instead?

@jywarren @cesswairimu @RuthNjeri

### PR DESCRIPTION
**Pull request by TildaDares**
Fixes #9732 (<=== Add issue number here)

Part of larger planning issue #9667

image

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Review by cesswairimu — APPROVED** [review]
Looks great to me :tada: thanks @TildaDares

**Comment by jywarren** [comment]
Agreed! Let's merge!

File: app/views/tag/_tags.html.erb [hunk 1]
```diff
@@ -11,7 +17,7 @@
...
+      <p class="badge <%= badge_name %> pop more-tags" style="display:none;cursor:pointer;margin-bottom:3px;" id="tag_<%= tag.tid %>" data-toggle="popover" data-trigger="focus" data-count=0 data-placement="top" data-content="<p style='text-align:center;'><a href='/tag/<%= tag.name %>'><%= Tag.tagged_node_count(tag.name) || 0 %> notes</a> - <a href='/contributors/<%= tag.name %>'><%= Tag.contributors(tag.name).count %> people <br></a></p><% if tag.location_tag? %><p>This is a geographical tag for placing this content on a map. Please visit our <a href='https://publiclab.org/wiki/location-privacy'>location privacy</a> page. </p><% end %> <p style='text-align:center;font-size:12px;'><%if tag.description %><%= tag.description %> |<% end %> created by <a href='/profile/<%= tag.try(:author).try(:username) %>'><%= tag.try(:author).try(:username) %></a> <%= time_ago_in_words(Time.at(tag.date)) %> ago </p><div class='text-center'><a href='/subscribe/tag/<%= tag.name %>' class='btn btn-primary'>Follow</a></div>" data-html="true" title="<%= tag.name %>">
```
Comment (jywarren) [inline app/views/tag/_tags.html.erb:20]: A couple ideas here, i think it's not possible in the attribute to break this onto multiple lines, but are there any other follow-ups we could do to make this more readable? 

Also, shall we try to put this text into the translation system? 

Great work!!

Comment (TildaDares) [reply app/views/tag/_tags.html.erb:20]: @jywarren The `data-content` attribute is still long but it's an improvement

```
<p 
  class="badge <%= badge_name %> pop more-tags"  
  style="display:none;cursor:pointer;margin-bottom:3px;" 
  id="tag_<%= tag.tid %>" 
  data-toggle="popover" 
  data-trigger="focus" 
  data-count=0 
  data-placement="top" 
  data-content="<p style='text-align:center;'><a href='/tag/<%= tag.name %>'><%= Tag.tagged_node_count(tag.name) || 0 %> notes</a> - <a href='/contributors/<%= tag.name %>'><%= Tag.contributors(tag.name).count %> people <br></a></p><% if tag.location_tag? %><p>This is a geographical tag for placing this content on a map. Please visit our <a href='https://publiclab.org/wiki/location-privacy'>location privacy</a> page. </p><% end %> <p style='text-align:center;font-size:12px;'><%if tag.description %><%= tag.description %> |<% end %> created by <a href='/profile/<%= tag.try(:author).try(:username) %>'><%= tag.try(:author).try(:username) %></a> <%= time_ago_in_words(Time.at(tag.date)) %> ago </p><div class='text-center'><a href='/subscribe/tag/<%= tag.name %>' class='btn btn-primary'>Follow</a></div>" 
  data-html="true" 
  title="<%= tag.name %>"
 >
```

I'll also work on putting the text into the translation system.

**Pull request merged**